### ISSUE TITLE
BUG: regression random state ICA

### ISSUE THREAD

**Issue by dengemann** [mne-tools/mne-python#2283]
In random_state == None will not any longer produce a fixed seed. I suspect some underlying changes in sklearn. @agramfort does this ring a bell?

**Comment by agramfort** [comment]
no bell

**Comment by dengemann** [comment]
turns out that our defaults == None will not produce a fixed seed. ...

**Comment by dengemann** [comment]
@agramfort we should maybe change that. This is confusing for users. Maybe just fixt the seed or print it.

**Comment by agramfort** [comment]
I would not fix it. Users should realize that ICA is a non-deterministic
algo.
I'd just improve the doc

**Comment by dengemann** [comment]
Yes, agreed. I think the confusion was arising from the fact that we indeed once had a bug but then fixed it. The examples then went fine as we passed a seed, not because of the defaults. I think I forgot about that.
